### DISCUSSION TITLE
Cannot match current PS1 prompt

### DISCUSSION THREAD

**Discussion by sherabi** [starship/starship#1562] [Q&A] [answered]
Hi,
As a way to test starships customizability, I am trying to match my current prompt which looks like the image attached. Is it possible to achieve such a prompt and what would my config look like? I tried quite a bit but nothing seems to work. Notice that in the image there are multiple symbols and there are no spaces which precede every module in startship.

<img width="413" alt="Screen Shot 2020-08-06 at 9 27 14 AM" src="https://user-images.githubusercontent.com/2041061/89537329-307a3300-d7c7-11ea-9c89-f9035aa19d77.png">

**Comment by chipbuster** [comment, marked as answer]
This will be possible in the next release, v0.45.0 (where the format strings change will be landing).

See #1454 and #1537 for details.
